orca-keel/orca-keel.gradle

```diff
@@ -1,25 +1,4 @@
-apply from: "$rootDir/gradle/kotlin.gradle"
-
-tasks.compileGroovy.enabled = false
-
 dependencies {
-  compile project(":orca-extensionpoint")
-  compile project(":orca-core")
-  compile project(":orca-front50")
-
-  compile "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${spinnaker.version('jackson')}"
-  compile "com.fasterxml.jackson.module:jackson-module-kotlin:${spinnaker.version("jackson")}"
-  compile('com.jayway.jsonpath:json-path:2.2.0')
-
-  compile spinnaker.dependency("slf4jApi")
-  compile spinnaker.dependency("bootAutoConfigure")
-  compile spinnaker.dependency("springContext")
-  compile spinnaker.dependency("jacksonDatabind")
-  compile spinnaker.dependency("spectatorApi")
-  compile spinnaker.dependency("frigga")
-  compile spinnaker.dependency('korkExceptions')
-
-  testCompile spinnaker.dependency("slf4jSimple")
-  testCompile 'org.spockframework:spock-unitils:1.1-groovy-2.4-rc-2'
-  testCompile 'org.codehaus.groovy:groovy-json:2.4.11'
+  compile project(":orca-retrofit")
+  testCompile project(":orca-test")
 }
```

Comment:
Most of this wasn't necessary.

Reply:
Thank you!

Reply:
We'll revisit the kotlin stuff later, maybe on Monday if you're feeling well.